tests/foreman/api/test_convert2rhel.py

```diff
@@ -316,8 +328,8 @@ def test_convert2rhel_centos(target_sat, centos, activation_key_rhel, version):
     host_content = target_sat.api.Host(id=centos.hostname).read_json()
     # workaround for BZ 2080347
     assert (
-        host_content['operatingsystem_name'].startswith(f"RHEL Server {version}")
-        or host_content['operatingsystem_name'].startswith(f"RedHat {version}")
-        or host_content['operatingsystem_name'].startswith(f"RHEL {version}")
+        host_content['operatingsystem_name'].startswith(f'RHEL Server {version}')
+        or host_content['operatingsystem_name'].startswith(f'RedHat {version}')
+        or host_content['operatingsystem_name'].startswith(f'RHEL {version}')
```

Comment:
Can you remove this workaround as the BZ mentioned is closed as wontfix and check only valid assertion?

Reply:
Updated.